### DISCUSSION TITLE
localhost unable to

### DISCUSSION THREAD

**Discussion by jlee88my** [WalletConnect/walletconnect-monorepo#3590] [Developer Support]
I have verified my domain, and it works. However, after verifying it, access from localhost is no longer allowed.

Refused to frame 'https://verify.walletconnect.com/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors https://*.eskrow.la https://eskrow.la".

From localhost, I now cannot use the w3modal anymore. I need to be able to do development on localhost. So, this is a show stopper for me :'(

How can I "unverify" my domain and allow localhost to function?

This is my current setting:
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/118353309/71f50046-7085-4b61-8a7d-08583dff61cb)

**Comment by glitch-txs** [comment]
Hey, thanks for reporting and sorry for the inconvenience, I'll send this to the team to take a look

**Reply by glitch-txs** [reply]
are you still able to open the modal? or scan the QR code?

**Reply by jlee88my** [reply]
Opps... I replied with a different post below. Thank you in advance for support :)

**Comment by jlee88my** [comment]
Yes, I am still able to open the modal, and connect. But once I "connect", it will show this message:
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/118353309/a405b092-5ca1-49e9-810b-b1b357b7246e)

Then becomes "Disconnected" again. The button will go back to "Connect Wallet"
** Note: This is from localhost. Everything works fine in https://eskrow.la

**Comment by rivaratyr** [comment]
Joining to this, same issue from localhost. "Refused to frame 'https://verify.walletconnect.com/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors [mydomains]."

<img width="724" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/97779276/00498283-62fb-40e5-a2ac-0d70d8f3450b">

Strange, because in WalletConnect app:
<img width="360" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/97779276/40752162-4c43-43b9-8b01-80e19c645110">

**Reply by rivaratyr** [reply]
It seems that the "Allowed Domains" settings is not working as intended. After removing all the entries, waiting for 15 minutes the above "Refused to frame 'https://verify.walletconnect.com/' because an ancestor violates the following Content Security Policy directive: "frame-ancestors" error message still persists.
